### DISCUSSION TITLE
Склонение

### DISCUSSION THREAD

**Discussion by konditerov** [totumonline/totum-ru-issues-and-discussions#267] [Есть идея, а что если...]
Столкнулся с проблемой, делаю договора и спецификации, нет склонений.
Возможно было бы хорошо подключить библиотеку для склонение ФИО, например https://github.com/petrovich

Конечно проблема гораздо глубже выяснилась, идеально подтягивать всю (нужную) инфу по организации через дадату + склонения (https://dadata.ru/api/#party)

**Comment by ttm-alexeypolunin** [comment]
Вы собственно можете вызвать dadata через getFromScript. Если речь идет о том, что бы это работало в момент редактирования поля, то такой функционал запланирован в pro, но из-за нулевого финансирования в нынешних условиях он неизвестно когда будет.

upd: в том числе такие api запросы рассматриваются как тарифицируемый коммерческий функционал через наш сервер

**Reply by konditerov** [reply]
Спасибо, попробую. Да, хотелось в момент редактирования поля.

Например в 1с есть кнопка "Заполнить по ИНН", попробую сделать такую-же, прикрутив скрипт на кнопку.
